generators/client/templates/common/.eslintignore.ejs

```diff
@@ -18,7 +18,12 @@
 -%>
 node_modules/
 src/main/docker/
+<%_ if (cypressTests) { _%>
+src/test/javascript/cypress/
```

Comment:
Should the entire cypress for be ignored?

Reply:
> I though it was merged to master 😆.

Not yet, but soon :p

I guess it can done when I'll review all your last suggestions.

Reply:
I think your are right, we should not ignore cypress at all (well I see no reason) but we if do that we need to add cypress folder in includes array in the tsconfig and fix all lint errors. I'll do that

Reply:
https://github.com/jhipster/generator-jhipster/pull/12309

Reply:
hum... it seems we did that because adding Cypress in the project (tsconfig by using include) fire errors and conflicts between libraries. The solution was to exclude Cypress to the tsconfig file and also add it to the eslint ignore to avoid errors like this:
The file does not match your project config [...]
he file must be included in at least one of the projects provided at [...]

Reply:
Agreed, this is not simple to fix.
A reference:
https://github.com/cypress-io/cypress-and-jest-typescript-example